### DISCUSSION TITLE
Best temperature sensor?

### DISCUSSION THREAD

**Discussion by scribblrsam** [Koenkk/zigbee2mqtt#8754] [Devices]
Looking for some advice, I have the tuya temp and humidity sensors and find that they are slow to update/react to changes. I know there are threads about this on here so didn't want to get into that unless there is an obvious fix that I've missed.

Can anyone recommend a good alternative?

Thanks

**Comment by c0mplex1** [comment]
Try the Konke temp and humidity sensor.
The minimal sample rate is every 30min.
https://www.zigbee2mqtt.io/devices/2AJZ4KPFT.html#konke-2ajz4kpft

![afbeelding](https://user-images.githubusercontent.com/81088928/133424737-3bf47ec2-3a52-434f-b9d2-7c1e3eb010c0.png)
